content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -228,7 +228,7 @@ kubectl rollout restart deployment/frontend                      # Rolling resta
 
 cat pod.json | kubectl replace -f -                              # Replace a pod based on the JSON passed into std
 
-# Force replace, delete and then re-create the resource. Will cause a service outage.
+# Force replaces, delete and then re-create the resource. Will cause a service outage.
```

Comment:
This sentence is not grammatically correct.  Could potentially change it to 
"Force replaces, deletes, and then re-creates the resource. Will cause a service outage."

But then the verb tense doesn't match the others so not sure its worth it